guides/source/configuring.md

```diff
@@ -1258,6 +1258,16 @@ development:
 
 Change the username and password in the `development` section as appropriate.
 
+#### Configuring Metadata Storage
+
+By default Rails will store information about your environment and schema in an internal table. If you do not want this table to be created, and are willing to give up the protections it provides (useful when working with a shared database and a database user that can not create tables).
```

Comment:
New guides should be wrapped to 80 characters. Also I think the guides can just say how to turn it on and don't need to mention giving up protections it provides.


```suggestion
By default Rails will store information about your Rails environment and schema in an internal
table named `ar_internal_metadata`.

To turn this off per connection, set `use_metadata_table` in your database configuration. This is
useful when working with a shared database and/or database user that cannot create tables.
```